cmake-docker-environment/labs/consume-shared-library/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.9.1)
+project(contumerTest)
+
+add_library(greeting greeting/greeting.cpp)
+target_include_directories(greeting PUBLIC ${CMAKE_CURRENT_LIST_DIR}/greeting/include)
+
+add_executable(consumer main.cpp)
+target_link_libraries(consumer greeting)
```

Comment:
All text files must end with LF

Reply:
How do I fix this?

Reply:
Just make sure in your editor that files have a last blank line